macros/src/openapi/mod.rs

```diff
@@ -202,16 +202,14 @@ fn quote_mediatype(m: &MediaType) -> Expr {
 }
 
 fn quote_schema_or_ref(ros: &ObjectOrReference<Schema>) -> TokenStream {
-    match ros {
-        ObjectOrReference::Ref { ref_path: r } => r
-            .parse::<TokenStream>()
-            .expect("failed to lex path to type"),
-        ObjectOrReference::Object(schema) => match &schema.ref_path {
-            r => r
-                .parse::<TokenStream>()
-                .expect("failed to lex path to type"),
-        },
-    }
+    let r#ref = match ros {
```

Comment:
I'm not sure if name of a variable worth raw identifier, but it's not a big issue I think.

Reply:
Yeah, I don't mind dropping raw identifier in favor of some other name. `ref_path`?

Reply:
Oh, I see that you've already merged it. Anyway, dutifully noted.